### PR TITLE
sensor: Add Silabs SI7055 support

### PR DESCRIPTION
**Pull request by greg-leach**
Add support for the Silabs SI7055 temperature sensor

Signed-off-by: Greg Leach <greg.leach@lairdconnect.com>

### CODE REVIEW

File: drivers/sensor/Kconfig [hunk 1]
```diff
@@ -149,6 +149,8 @@ source "drivers/sensor/qdec_nrfx/Kconfig"
 
 source "drivers/sensor/sht3xd/Kconfig"
 
+source "drivers/sensor/si7055/Kconfig"
+
 source "drivers/sensor/si7006/Kconfig"
```
Comment (rerickson1) [inline drivers/sensor/Kconfig:155]: should si7055 come after si7006 to keep with alphabetical order?

**Comment by greg-leach** [comment]
More merge issues.

**Comment by MaureenHelm** [comment]
> More merge issues.

Please force push to your branch to update the PR instead of opening a new PR.

**Comment by greg-leach** [comment]
Hi Maureen – I unfortunately cannot re-open the Pull Request due to having deleted and re-cloning my fork.

Greg.

**Pull request closed without merging**